resource_estimator/src/estimates/stages/physical_estimation/tests.rs

```diff
@@ -264,13 +296,12 @@ pub fn test_hubbard_e2e_measurement_based() -> super::super::super::Result<()> {
     validate_result_invariants(&result);
 
     let output_t_error_rate = result
-        .required_logical_tstate_error_rate()
+        .required_logical_magic_state_error_rate()
```

Comment:
should it be magic state, factory or tfactory?

Reply:
magic state, because the output rate of a factory can be different, if it produces multiple states.